### PR TITLE
add shuttle bus markers (issue #39). They are now displayed on the ma…

### PR DESCRIPTION
**Pull request by razine-bensari**
…p with a concordia icon different from all buildings markers to differentiate both for all users. Test are implemented and run and pass.

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -65,11 +65,11 @@ class App extends Component<{}, appState> {
...
   }
 
-  setMapLocation = (location: Location) => {
+  setMapLocation = (latitude, longitude) => {
     this.setState({
       region: {
-        latitude: location.getLatitude(),
-        longitude: location.getLongitude(),
+        latitude,
+        longitude,
```
Comment (Alessandro100) [inline App.tsx]: Why was this changed, location object should be the standard

File: constants/CampusShuttleBusStop.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const ShuttleBusMarkers = [
+  {
+    coordinate: {
+      latitude: 45.49702,
+      longitude: -73.578519,
+    },
+    title: 'BUS STOP SGW',
+    description: 'Shuttle Bus Stop Location for SGW campus',
+    pinColor: 'rgb(42,65,75)',
```
Comment (Alessandro100) [inline constants/CampusShuttleBusStop.tsx]: Can you add all colors in the colors constant file

File: App.tsx (hunk 1 above)
Comment (razine-bensari) [reply App.tsx]: I have no idea, Ill add it back. I did not touch this file ;.<
